src/app/+item-page/edit-item-page/item-status/item-status.component.ts

```diff
@@ -78,42 +78,36 @@ export class ItemStatusComponent implements OnInit {
         The value is supposed to be a href for the button
       */
       const operations = [];
-      operations.push(new ItemOperation('authorizations', this.getCurrentUrl(item) + '/authorizations'));
-      operations.push(new ItemOperation('mappedCollections', this.getCurrentUrl(item) + '/mapper'));
-      operations.push(undefined);
-      // Store the index of the "withdraw" or "reinstate" operation, because it's added asynchronously
-      const indexOfWithdrawReinstate = operations.length - 1;
+      operations.push(new ItemOperation('authorizations', this.getCurrentUrl(item) + '/authorizations', FeatureID.CanManagePolicies, true));
+      operations.push(new ItemOperation('mappedCollections', this.getCurrentUrl(item) + '/mapper', FeatureID.CanManageMappings, true));
+      if (item.isWithdrawn) {
+        operations.push(new ItemOperation('reinstate', this.getCurrentUrl(item) + '/reinstate', FeatureID.ReinstateItem, true));
+      } else {
+        operations.push(new ItemOperation('reinstate', this.getCurrentUrl(item) + '/reinstate', FeatureID.WithdrawItem, true));
```

Comment:
This doesn't look correct. If the Item is **not withdrawn**, then the button should allow for Withdrawing the Item.  Currently, on the Edit Item page in this PR, I always see a "Reinstate" button (never "Withdraw") and I think this is why.

Reply:
The withdraw button and `canManageBitstreamBundles` feature issues should now be fixed!